tests/utils.py

```diff
@@ -1,27 +1,18 @@
 import os
-import pytest
 from glob import glob
 
 
-def skip_func(msg):
-    def test_func(*args, **kwargs):
-        pytest.skip(msg)
-    return test_func
-
```

Comment:
I had to change that because of it ignores parametrized tests, which means in some job you'd get
```
tests/test_threadpoolctl.py::test_openmp_nesting[None] PASSED
tests/test_threadpoolctl.py::test_openmp_nesting[1] PASSED
tests/test_threadpoolctl.py::test_openmp_nesting[2] PASSED
tests/test_threadpoolctl.py::test_openmp_nesting[4] PASSED
```
and in other jobs you'd get
```
tests/test_threadpoolctl.py::test_openmp_nesting SKIPPED
```
which makes merging all test files painful.

Using skipif makes pytest skip the tests individually.